cli/azd/pkg/input/console.go

```diff
@@ -133,15 +143,16 @@ type AskerConsole struct {
 }
 
 type ConsoleOptions struct {
-	Message      string
-	Help         string
-	Options      []string
-	DefaultValue any
+	Message string
+	Help    string
+	Options []string
 
-	// Prompt-only options
+	// OptionDetails is an optional field that can be used to provide additional information about the options.
+	OptionDetails []string
```

Comment:
For azd's internal consumption, I have slight preference for tying the title and description for each option just to avoid index-based mishaps. I do think it's a good idea to still keep the simplicity of `[]string` around. On my initial read, OptionDetails did jump out to me as "slightly unexpected".

```golang
type ConsoleOptions struct {
  // Options to select from.
  Options []string
  // A longer version of Options that can provide lengthy descriptions.
  // It is valid to set only one of Options or LongOptions.
  LongOptions []LongOption
}

type LongOptions struct {
  Title string
  Description string
}
```